### PR TITLE
[qwt] update to 6.1.5

### LINKED ISSUE #12336: [qwt] update to 6.1.5
Library name: qwt

New version number: 6.1.5

> Qwt 6.1.5 adds some necessary includes to be compilable with Qt 5.15

* Changelog: https://sourceforge.net/p/qwt/code/HEAD/tree/tags/qwt-6.1.5/CHANGES-6.1
* News: https://sourceforge.net/p/qwt/news/2020/06/qwt-615/
* Source: https://sourceforge.net/projects/qwt/files/qwt/6.1.5/

### PR DESCRIPTION
**Pull request by JonLiu1993**
Fix #12336
 
Update qwt to the latest version 6.1.5

All features have passed with the following triplets:

- x86-windows
- x64-windows
- x64-windows-static

### CODE REVIEW

File: ports/qwt/build.patch [hunk 1]
```diff
@@ -1,130 +1,129 @@
-diff --git a/src/qwt_compass_rose.cpp b/src/qwt_compass_rose.cpp
-index 8acb045c2..b7b676712 100644
---- a/src/qwt_compass_rose.cpp	
-+++ b/src/qwt_compass_rose.cpp
-@@ -11,6 +11,7 @@
- #include "qwt_point_polar.h"
+diff --git a/src/qwt_compass_rose.cpp b/src/qwt_compass_rose.cpp
```
Comment (Neumann-A) [inline ports/qwt/build.patch:1]: Have you tried to build qwt without the patch? I thought the sole reason to update is to fix exactly this issue.

Comment (JonLiu1993) [reply ports/qwt/build.patch:1]: I'm really sorry, thank you for reminding me, I successfully rebuilt without using patches.I will resubmit.

**Review by NancyLi1013 — APPROVED** [review]

**Review by BillyONeal — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks!

**Pull request merged**